browser/extensions/brave_wallet_navigation_throttle_unittest.cc

```diff
@@ -80,6 +81,12 @@ class BraveWalletNavigationThrottleUnitTest
 #endif
     original_client_ = content::SetBrowserClientForTesting(&client_);
     content::RenderViewHostTestHarness::SetUp();
+
+    // For debug builds, set a fake BRAVE_INFURA_PROJECT_ID env var
+    std::unique_ptr<base::Environment> env(base::Environment::Create());
```

Comment:
I am not using the new util inside the unit test as `BRAVE_INFURA_PROJECT_ID` is not available in test envs for now